here-naksha-lib-hub/src/main/java/com/here/naksha/lib/hub/NakshaHub.java

```diff
@@ -300,7 +300,8 @@ private static WriteXyzCollections createAdminCollectionsRequest() {
     return new ExtensionConfig(
         System.currentTimeMillis() + extensionConfigParams.getIntervalMs(),
         extList,
-        extensionConfigParams.getWhiteListClasses());
+        extensionConfigParams.getWhiteListClasses(),
+        this.nakshaHubConfig.env);
```

Comment:
print env as `this.nakshaHubConfig.env.toLowerCase()` , as that is how we are using it while loading Jar rom S3

Reply:
fixed